### ISSUE TITLE
gallery

### ISSUE THREAD

**Issue by vai69** [Walchand-Linux-Users-Group/wcewlug.org#23]
pics differentiated according to events

**Comment by SakhawalkarMaithili** [comment]
** Apply  a related image for every event on the main page of gallery, instead of using the same for all .

**Comment by Palak-shravagi** [comment]
Component vise distribution
|
+--Header
|
+--Gallery (Event wise gallery card)
|      |
|      +--Open source
|      |
|      +--Metamorphosis ....
|
+-- Footer

**Comment by suyash-chavan** [comment]
> ** Apply a related image for every event on the main page of gallery, instead of using the same for all .

Yes seperate page will be good for each event and only one image on main gallery page.

**Comment by suyash-chavan** [comment]
@Palak-shravagi Assign kela nahi ka?

**Comment by Palak-shravagi** [comment]
abhi krti hu assign
